clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerService.java

```diff
@@ -577,9 +577,11 @@ private Map<PubSubTopicPartition, TopicPartitionIngestionInfo> getIngestionInfoF
         double byteRate = partitionStats.getBytesRate();
         long lastSuccessfulPollTimestamp = partitionStats.getLastSuccessfulPollTimestamp();
         long elapsedTimeSinceLastPollInMs = ConsumptionTask.DEFAULT_TOPIC_PARTITION_NO_POLL_TIMESTAMP;
```

Comment:
Maybe the naming here is a bit confusing for `elapsedTimeSinceLastPollInMs` and `elapsedTimeSinceLastPolledRecordsInMs`, maybe we could change `elapsedTimeSinceLastPollInMs` and ``elapsedTimeSinceLastPolledRecordsInMs``to `elapsedTimeSinceLastConsumerPollInMs` and `elapsedTimeSinceLastRecordForPartitionInMs`, what do you think? Sorry, my ticket initially does not give a good instruction.  Also consider to add java doc to these fields.